docker/settings.py

```diff
@@ -19,3 +19,39 @@
         '/gcs/credentials.json')
 
 AWS_STORAGE_BUCKET_NAME = os.getenv('AWS_STORAGE_BUCKET_NAME', '')
+
+# Settings that control student data provisioning
+INCLUDED_CLASS_CODES = ['1', '2', '3', '4', '5', '8']
+INCLUDED_CAMPUS_CODES = ['0', '1']
+EXCLUDED_MAJOR_CODES = ['0-EMBA', '0-GEMBA']
+ENGR_COLLEGE_MAJORS = ['0-BIOEN', '0-BSE', '0-DATA', '0-PHARBX', '0-PREBSE',
+                        '0-TECH I', '0-C SCI', '0-CMP E', '0-CSE']
+
+"""
+maps fields of the expected handshake fields to the fields as they appear in the
+student data
+
+adding '[]' after an attribute indicates that it is a list of values, and an
+index inside means to only grab that index of the attribute list
+"""
+HANDSHAKE_FIELDNAMES = {
+    'auth_identifier': 'uwnetid',
+    'username': 'uwnetid',
+    'email_address': 'student:student_email',
+    'card_id': 'student:student_number',
+    'school_year_name': 'student:class_code',
+    'campus_name': 'student:campus_code',
+    'last_name': 'last_name',
+    'first_name': 'first_name',
+    'middle_name': 'preferred_middle_name',
+    'preferred_name': 'preferred_first_name',
+    'primary_education:college_name': 'student:majors[]:college',
+    'primary_education:major_names[]': 'student:majors[]:major_name',
+    'primary_education:primary_major_name': 'student:majors[0]:major_name',
+    # 'primary_education:education_level_name': '',
```

Comment:
These are commented out because they currently don't seem to have a person client data equivalent